src/common/transport/uart_transport.cpp

```diff
@@ -283,6 +288,26 @@ struct UartTransport::impl : Transport
             }
 #endif
 
+#if defined(_WIN32)
+            ::COMMTIMEOUTS timeouts;
+
+            // See
+            // https://docs.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-commtimeouts
+            // for documentation of these parameters
+            timeouts.ReadIntervalTimeout        = MAXDWORD;
+            timeouts.ReadTotalTimeoutMultiplier = MAXDWORD;
+            timeouts.ReadTotalTimeoutConstant   = readTotalTimeoutConstant;
+
+            timeouts.WriteTotalTimeoutMultiplier = 0;
+            timeouts.WriteTotalTimeoutConstant   = 0;
+
+            if (!::SetCommTimeouts(serialPort->native_handle(), &timeouts))
```

Comment:
Is the behaviour defined for Linux/OsX when this SetCommTimeouts is not set?

Reply:
No, they are not. But the defaults used are good enough for the customer.